source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -0,0 +1,170 @@
+==============
+Retrieve Data
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to retrieve data from your MongoDB
+collection using **read operations**.
+
+Read operations allow you to do the following: 
+
+- Retrieve a subset of documents from your collection by using a :ref:`find operation <retrieve-find>` 
+- Perform transformations on retrieved documents from your collection using the :ref:`aggregate operation <retrieve-aggregate>`. 
+
+Sample Data
+~~~~~~~~~~~
+
+To follow along with the examples on this page, run the following
+snippet to load the documents into the ``ratings`` collection in the
+``tea`` database:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/retrieve.go
+   :language: go
+   :dedent:
+   :start-after: begin insert docs
+   :end-before: end insert docs
+
+Each document contains the tea a person tried and their rating on a
+scale of 10, which corresponds to the ``name`` and ``rating`` fields.
+
+.. note::
+
+   Each example truncates the ``ObjectID`` value since the driver
+   generates them uniquely. 
+
+.. _retrieve-find:
+
+Find operations
+---------------
+
+Use a **find operation** to retrieve a subset of your existing data.
+Find operations consist of the ``Find()`` and ``FindOne()`` functions.
+
+Find All Documents
+~~~~~~~~~~~~~~~~~~
+
+The ``Find()`` function expects you to pass a ``Context`` type and a
+filter. The function returns *all* documents that match the filter as a
+``Cursor`` type. If you pass an empty filter, the function returns *all*
+documents in the collection. 
+
+.. To learn how to access data in a cursor, see the :doc:`Cursor
+.. </fundamentals/crud/read-operations/cursor>` guide. 
+
+Find One Document
+~~~~~~~~~~~~~~~~~
+
+The ``FindOne()`` function expects you to pass a ``Context`` type and a
+filter. The function returns *the first document* that matches the
```

Comment:
nit: I believe as a team we've decided to use the term `query filter`. It's okay to use `filter` in other instances as long as we state the full name at the beginning of the paragraph imo.
```suggestion
query filter. The function returns *the first document* that matches the
```